mangohero1985/devops
# first merge

* finished:
1. customized databricks docker image
2. unittest on python methods and pyspark methods
3. add slack notification by circleci
4. add coverage check in cicleci
5. link with codecov
